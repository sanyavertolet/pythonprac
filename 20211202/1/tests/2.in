class A(metaclass=dump):
    def moh(self, seemtobe='moh'):
        return f'Become something that seems to be {seemtobe}'

a = A()
print(a.moh())
print(a.moh('moh?'))

