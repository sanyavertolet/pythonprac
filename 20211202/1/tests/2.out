moh: (), {}
Become something that seems to be moh
moh: ('moh?',), {}
Become something that seems to be moh?
